framework/db/sqlite/Schema.php

```
@@ -479,6 +564,91 @@ private function loadTableConstraints($tableName, $returnType)
+        $quoteCharacter = null;
+        for ($i = 0, $length = strlen($name); $i < $length; $i++) {
+            $character = $name[$i];
+            if ($quoteCharacter !== null) {
+                if ($character !== $quoteCharacter) {
+                    $part .= $character;
+                } elseif (isset($name[$i + 1]) && $name[$i + 1] === $quoteCharacter) {
+                    $part .= $character;
+                    $i++;
+                } else {
+                    $quoteCharacter = null;
+                }
```

PHPMD (2.15.0)

[error] The method getTableNameParts uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)